cookbook/form/dynamic_form_modification_ajax_js.rst.inc

```diff
@@ -0,0 +1,25 @@
+<script>
+var $sport = $('#meetup_sport');
```

Comment:
we can write another value here $('#meetup_sport'); such $('#meetup'); or we must specified the id in the template??